object_status.php

```diff
@@ -33,13 +33,9 @@
 use tool_objectfs\local\report\object_status_history_table;
 
 $reportid = optional_param('reportid', 0, PARAM_INT);
```

Comment:
what happens when the param is missing?

Reply:
Then `$reportid` is set ot 0. And later set to the latest report snapshot `$reportid = key($reports);`